src/components/language/__tests__/radiobuttons.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Radio from "../radiobuttons";
+
+const onChange = jest.fn();
+const language = "english";
```

Comment:
Considering that these two variables are only used in one test, I would probably move them within that test.